src/routes/home/HomeScreen.tsx

```diff
@@ -56,6 +60,21 @@ const HomeScreen = ({ navigation }: HomeScreenProps): JSX.Element => {
     })
   }
 
+  const navigateToNextExercise = async (item: Discipline): Promise<void> => {
+    getNextExercise(item)
+      .then(async nextExerciseData => {
```

Comment:
I think it is not good to mix async/await inside then/catch blocks. Instead, you should either chain it correctly or only use async/await style syntax.